lib/pbench/test/unit/server/test_requests.py

```diff
@@ -201,38 +212,43 @@ def test_invalid_authorization_upload(client, caplog, server_config):
 
             # Upload with invalid token
             response = client.put(
-                f"{server_config.rest_uri}/upload/ctrl/{socket.gethostname()}",
+                TestUpload.gen_uri(server_config),
                 headers={
                     "Authorization": "Bearer " + auth_token,
+                    "Content-Length": "42",
```

Comment:
But, wait; isn't 42 __always__ the right answer???

Reply:
Yes, but the confounding `test_client()` ignores the right answer and does what it wants only!